### ISSUE TITLE
facebookbot user creation error :  Whoami endpoint returned M_FORBIDDEN

### ISSUE THREAD

**Issue by jaalmoes** [tulir/mautrix-facebook#82]
Hi, 
I have followed the wiki but when running python -m mautrixe_facebook I ran into this issue: 

It seems like the programme can't register the facebookbot user but I don't know why. I can register users manually using the command :
register_new_matrix_user -c homeserver.yaml http://localhost:8008

I don't know if this is a configuration issue or not, I am sorry if it is. 

Thank you

(messenger) ircd@shirocookie:~/messenger/lib/python3.8/site-packages/mautrix_facebook$ python -m mautrix_facebook 
[2020-07-15 17:36:16,956] [INFO@mau.init] Initializing mautrix-facebook 0.1.0rc3
[2020-07-15 17:36:16,975] [INFO@mau.init] Initialization complete in 0.12 seconds
[2020-07-15 17:36:16,976] [DEBUG@mau.init] Running startup actions...
[2020-07-15 17:36:16,976] [DEBUG@mau.init] Starting appservice...
[2020-07-15 17:36:16,976] [DEBUG@mau.as] Starting appservice web server on 0.0.0.0:29319
[2020-07-15 17:36:16,978] [INFO@mau.mx] Ensuring connectivity to homeserver
[2020-07-15 17:36:17,640] [DEBUG@mau.mx] Whoami endpoint returned M_FORBIDDEN, trying to register bridge bot before retrying...
[2020-07-15 17:36:17,648] [CRITICAL@mau.init] Unexpected error in main event loop
Traceback (most recent call last):
  File "/home/ircd/messenger/lib/python3.8/site-packages/mautrix/bridge/matrix.py", line 75, in wait_for_connection
    await self.az.intent.whoami()
  File "/home/ircd/messenger/lib/python3.8/site-packages/mautrix/client/api/authentication.py", line 32, in whoami
    resp = await self.api.request(Method.GET, Path.account.whoami)
  File "/home/ircd/messenger/lib/python3.8/site-packages/mautrix/api/http.py", line 203, in request
    return await self._send(method, endpoint, content, query_params, headers or {})
  File "/home/ircd/messenger/lib/python3.8/site-packages/mautrix/api/http.py", line 143, in _send
    raise make_request_error(http_status=response.status,
mautrix.errors.request.MForbidden: Application service has not registered this user

**Comment by bertkdowns** [comment]
I just had this issue. I think the problem is that the matrix identity servers are saying the bridge bot doesn't exist (probably because they're looking in the wrong place), when it does. The script then tries to create the bot but it doesn't work because it is already created.
So for me, since my server is running on port 8448, I had to change the config.yaml from:
`homeserver:
    # The address that this appservice can use to connect to the homeserver.
    address: https://example.com:8448
    # The domain of the homeserver (for MXIDs, etc).
    domain: example.com
`

to 

`homeserver:
    # The address that this appservice can use to connect to the homeserver.
    address: https://example.com:8448
    # The domain of the homeserver (for MXIDs, etc).
    domain: example.com:8448                            <------ THIS BIT CHANGED
`
TL:DR play around with the homeserver values

**Comment by tulir** [comment]
Debugging setup problems should be done in the Matrix room ([#facebook:maunium.net](https://matrix.to/#/#facebook:maunium.net))

**Comment by HammyHavoc** [comment]
I seem to be having this issue, but nobody has had any thoughts on https://matrix.to/#/#facebook:maunium.net: https://matrix.to/#/!JiBnIoyxTdNPsqbMwu:maunium.net/$XCxnrbdaAQ3bmixeGpTsZdXcK7jaGoNIjsh-Pmbr82U?via=matrix.org&via=matrix.splitanatom.com&via=maunium.net
